My_GOT_app/app/src/main/java/com/example/my_got_app/StarkActivity.java

```diff
@@ -45,21 +54,27 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     /**
-     *
+     * Initializes the Grid Layout using RecyclerView
+     * Initializes an ArrayList called starkHouse using prepareList
+     * Sets the list adapter
      */
     private void initViews(){
         RecyclerView recyclerView = (RecyclerView)findViewById(R.id.card_recycler_view);
         recyclerView.setHasFixedSize(true);
         RecyclerView.LayoutManager layoutManager = new GridLayoutManager(getApplicationContext(),2);
         recyclerView.setLayoutManager(layoutManager);
 
-        ArrayList<Person> starkHouse = prepareData();
+        ArrayList<Person> starkHouse = prepareList();
         ListAdapter adapter = new ListAdapter(starkHouse, getApplicationContext());
         recyclerView.setAdapter(adapter);
         Log.d(TAG,"init views" );
     }
 
-    private ArrayList<Person> prepareData(){
+    /**
+     * Sets the data in a list
+     * @return an ArrayList containing a Person object for each family member
+     */
+    private ArrayList<Person> prepareList(){
```

Comment:
This is a code duplicate - could have been written once (receiving the needed data as parameters).
See the comments I've left in LannisterActivity.